math-dojo/single-page-web-app
# Add question authoring page

## Purpose

* Adds a webpage to the UI for authoring questions
* adds form validation to the web page




## Does this introduce a breaking change?

```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[X] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code
npm start
go to link http://localhost:4200/create
check that the web page renders and form validation works
